ccmlib/repository.py

```diff
@@ -327,10 +335,14 @@ def download_version(version, url=None, verbose=False, binary=False):
     """
     assert_jdk_valid_for_cassandra_version(version)
 
+    u = ARCHIVE
```

Comment:
What do you think about renaming `u` => `url`?

Reply:
There is already a `url` kwarg to the function,  I can come up with a better name though.